src/egon/data/datasets/power_plants/pv_rooftop_buildings.py

```diff
@@ -2797,6 +2801,14 @@ def pv_rooftop_to_buildings():
     cap_per_bus_id_df = pd.DataFrame()
 
     for scenario in SCENARIOS:
+        if scenario == "status2019":
+            desagg_mastr_gdf = desagg_mastr_gdf[
+                desagg_mastr_gdf.Inbetriebnahmedatum < "2020-01-01"
+            ]
+            scenario_buildings_gdf = scenario_buildings_gdf[
+                scenario_buildings_gdf.Inbetriebnahmedatum < "2020-01-01"
+            ]
+
```

Comment:
just a style thing - there is actually a value we can refer to in the datasets.yml so I would rather code it similar to this: https://github.com/openego/powerd-data/blob/11a8348d7da0f40aae082df2ca3cc8253c87c2e9/src/egon/data/datasets/power_plants/mastr.py#L319-L322

Reply:
I changed it according to my comment. Are you fine with it  @ClaraBuettner ?